src/index.js

```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import './bootstrap.min.css';
+
+const allTasks = [
+  {
+    description: 'Configure webpack',
+    completed: false,
+  },
+  {
+    description: 'Go to college',
+    completed: true,
+  },
+  {
+    description: 'Request for a review',
+    completed: false,
+  },
+];
```

Comment:
- [ ] As specified in the project requirement kindly consider adding an `index` for each object of the array such that you can identify each todo task by its id and can also remove tasks in the future when you implement that functionality. See below for more details:
<img width="1003" alt="Screenshot 2023-06-07 at 07 03 36" src="https://github.com/M-Anwar-Hussaini/To-Do-List/assets/107192076/b3d630d1-b0c3-435f-9e86-b7b65e887f33">